tb_plugin/torch_tb_profiler/profiler/data.py

```diff
@@ -120,20 +123,51 @@ def process(self):
 
         logger.debug("OverallParser")
         overall_parser = OverallParser()
-        overall_parser.parse_events(self.events, module_parser.runtime_node_list, module_parser.device_node_list)
+        overall_parser.parse_events(self.events, module_parser.runtime_node_list, module_parser.device_node_list, module_parser.communication_data)
```

Comment:
It would be better if we could introduce one class like ProfileContext which includes all the node data like: cpp_op_list, kernel_list, runtime_node_list, device_node_list, communication_data. In this way, we can pass the context to both ModuleParser and OverrallParser without needing change the parse_event every time adding one view.

Reply:
thanks, done